manifest.md

```diff
@@ -85,13 +85,20 @@ A client will distinguish a manifest list from an image manifest based on the Co
 
     This OPTIONAL property contains arbitrary metadata for the manifest list.
     Annotations is a key-value, unordered hashmap.
-    Keys are unique, and best practice is to namespace the keys.
+    Keys MUST be unique, and best practice is to namespace the keys.
+	Keys starting with the `opencontainers.org` namespace are reserved and MUST NOT be used.
```

Comment:
On Mon, Jul 18, 2016 at 10:31:59AM -0700, Vincent Batts wrote:
> > @@ -85,13 +85,20 @@ A client will distinguish a manifest list from an image manifest based on the Co
> >  
> >      This OPTIONAL property contains arbitrary metadata for the manifest list.
> >      Annotations is a key-value, unordered hashmap.
> > -    Keys are unique, and best practice is to namespace the keys.
> > +    Keys MUST be unique, and best practice is to namespace the keys.
> > +	Keys starting with the `opencontainers.org` namespace are reserved and MUST NOT be used.
> 
> for all the specifics of clarification, it's interesting you don't
> see a difference in the two

I'm all for picking one ordering to recommend, but I'm happy to flip a
coin to pick ;).  Does anyone have a reason to *not* choose via
coin-flip?

It also looks like there's inconsistent whitespace on the indent here,
with the final new line using a tab while everyone else uses spaces.